2/debian-10/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -230,6 +231,23 @@ ldap_add_schemas() {
     debug_execute ldapadd -Y EXTERNAL -H "ldapi:///" -f "${LDAP_CONF_DIR}/schema/nis.ldif"
 }
 
+########################
+# Add custom schema
+# Globals:
+#   LDAP_*
+# Arguments:
+#   None
+# Returns
+#   None
+#########################
+ldap_add_custom_schema() {
+    info "Adding custom Schema : $LDAP_CUSTOM_SCHEMA_FILE ..."
+    debug_execute slapadd -F "$LDAP_ONLINE_CONF_DIR" -n 0 -l  "$LDAP_CUSTOM_SCHEMA_FILE"
```

Comment:
Could that be 

```suggestion
    debug_execute for f in $LDAP_CUSTOM_SCHEMA_DIR/*.ldif; do slapadd -F "$LDAP_ONLINE_CONF_DIR" -n 0 -l $f; done
```

And that env variable to be a directory instead of just single file. In same fashion as the LDAP_CUSTOM_LDIF_DIR.

Reply:
better indeed to support multiple custom schemas